cicd-scripts/install-dependencies.sh

```diff
@@ -7,11 +7,11 @@ echo "install dependencies"
 _OPERATOR_SDK_VERSION=v1.4.2
 
 if ! [ -x "$(command -v operator-sdk)" ]; then
-    if [[ "$OSTYPE" == "linux-gnu" ]]; then
-            curl -L https://github.com/operator-framework/operator-sdk/releases/download/${_OPERATOR_SDK_VERSION}/operator-sdk_linux_amd64 -o operator-sdk
-    elif [[ "$OSTYPE" == "darwin"* ]]; then
-            curl -L https://github.com/operator-framework/operator-sdk/releases/download/${_OPERATOR_SDK_VERSION}/operator-sdk_darwin_amd64 -o operator-sdk
-    fi
-    chmod +x operator-sdk
-    sudo mv operator-sdk /usr/local/bin/operator-sdk
+  if [[ $OSTYPE == "linux-gnu" ]]; then
+    curl -L https://github.com/operator-framework/operator-sdk/releases/download/${_OPERATOR_SDK_VERSION}/operator-sdk_linux_amd64 -o operator-sdk
+  elif [[ $OSTYPE == "darwin"* ]]; then
+    curl -L https://github.com/operator-framework/operator-sdk/releases/download/${_OPERATOR_SDK_VERSION}/operator-sdk_darwin_amd64 -o operator-sdk
+  fi
```

Comment:
could probably make this a one liner with some uname command as a subshell?

Reply:
Yup! Just fyi, these shellscript changes are from formatting, not explicit changes

Reply:
yep, i get that, just flagging as I go but will pause on that and we can tackle bash maybe in isolation in a follow up